components/Modal.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef, useState } from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+
+const Modal = ({ show, onClose, children, title }) => {
+  const [isBrowser, setIsBrowser] = useState(false);
+  useEffect(() => {
+    setIsBrowser(true);
+  }, []);
+
+  const handleCloseClick = (e) => {
+    e.preventDefault();
+    onClose();
+  };
+
+  const modalContent = show ? (
+    // <div className={styled.overlay}>
```

Comment:
Comments are good for yourself, but not in the main branch, so please get rid of it.

Reply:
i tried to used modal but i changed method of push a new reviews, i deleted the modal and all of his methods and components